package.json

```diff
@@ -74,7 +75,6 @@
     "rollup": "^2.36.1",
     "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-postcss": "^4.0.0",
```

Comment:
This was complaining, it needed a newer version of postCSS after I added less or somthing. Are we using it or just copied from another packege.json?

Reply:
Don't think you should use less/post-css here, instead rely in css-in js like makeStyles or however the Theme thing works.

Reply:
Postcss is used in nebula, so probably where the warning comes from.